### DISCUSSION TITLE
Kick-off Meeting for TriliumNext Organization

### DISCUSSION THREAD

**Discussion by root-hal9000** [TriliumNext/Notes#187] [Announcements]
We have set a time for a meeting to discuss the organization - this was suggested in discussion #2, but I am posting the details here for visibility. 

## WHEN:
**Sunday, July 7th, at 5:00 PM UTC**

## WHERE:
We are using Jitsi for this meeting. You can join with your browser, or mobile apps for Android ([F-Droid](https://f-droid.org/en/packages/org.jitsi.meet/) or [Google Play](https://play.google.com/store/apps/details?id=org.jitsi.meet)) and [iOS](https://apps.apple.com/us/app/jitsi-meet/id1165103905)
Here's the meeting link:
https://meet.jit.si/moderated/260e53a5d58765ec8ba11c60fc02566dc8b85b07a0f5f90c8f5c4fba2a1e2d22

Hope to see you there!

@TriliumNext/everyone 

## PROPOSED AGENDA

This is the current agenda for the meeting, I have looked through several discussions, chats, etc to compose it based on ideas, questions, concerns, and comments made by members of this community - and tried my best to include any proposed ideas.  Please let me know if you would like to raise other issues to be included in this agenda! I will edit this draft to include anything else I might have missed. We are aware that not all can make it - I will try my best to take extensive notes to report back to the community

Please keep in mind that this is only an initial meeting, and it is unlikely that we can get into very granular details. Out of respect for people's times, we are aiming to keep it between 60 to 90 minutes at most - I will do my best to moderate/direct discussion with that goal in mind

If you are on any of the teams, and could speak for a quick few minutes to give a report on what the team needs or what it is working on, please comment to let me know - I added myself as a volunteer to speak for the docs team.

----------------

1. ### **Introductions**
    1.  Go around the table for anyone who wishes to speak up and introduce themselves
        1.  If you would like, please let us know about why you joined and how you would like to contribute
        2.  Please let us know what teams you're a part of
    2.  Identify/Acknowledge other key contributors who are not present
        1.  Raise a toast for Zadam!
2. ### **Team Status Reports:**   
    Have one person per team speak (**briefly**) about what they are working on, future plans, needs/questions they have for the broader organization
    1.  Translations (speaker: volunteer?)
    2.  Issue Triage (speaker: volunteer?)
    3.  Documentation (speaker: root@HAL9000)
        1.  Documentation review project & documentation standards
        2.  Coordination with devs on ensuring that future changes are documented
        3.  Plans for ensuring code documentation 
    4.  Testing (speaker: volunteer?)
    5.  Devs (speaker: volunteer?)
3. ### **The Organization**
    1.  #### Name
        1.  Proposal:  A vote on the official name for the organization. If we can reach an unanimous decision, we can decide at the meeting, if not, then a later vote to the whole organization. Current suggestions:
            1.  TrilumNext (we already own a triliumnext.com domain name)
            2.  NeoTrilium  (we already own a neotrilium.org domain name)
            3.  Quadrilium
    2. ####  Organization Principles (potential commitments for a future charter)
        1.  Proposal: The organization makes a public commitment that no feature of the software shall be restricted or paywalled regardless of any future decisions on funding
        2.  Transparency: do we need any kind of specific rules regarding public reporting regarding organizational decisions, funding, votes, meeting minutes, etc 
    3. ####  Governance
        1.  Proposals regarding forming a governing body (executive board) with the authority to make decisions for the organization. The following proposals are gathered from ongoing discussions among members, and aim to create a simple, lean organizational structure, which can be later refined as needed 
            1.  Size: 
                1.  Proposal: Executive board should consist of 5 members. If other size, should be an odd number for voting purposes
            2.  Decision making
                1.  Proposal: A board vote can be called for any major decision with serious impact to the organization. A simple majority is required on any given decision. Assuming 5 members - Any proposal always has to be communicated to all board members - If simple majority is already agreed, there is no need to wait for other members to vote. If not, there shall be a period of at least 7 days waiting for the input of other members before the decision can be finalized without their input
            3.  Board Membership proposals:
                1.  Public call for volunteers among existing members. If we have only 5 or less volunteers, no vote is needed. If more, we can have a discussion on how to select board members and/or come to an agreement on rotating membership.
                2.  Terms of 6 months, renewable if there are no other volunteers.
                3.  Removal: A board remember can be removed for any reason (such as abandonment or malicious actions) by unanimous vote of all other board members.
            4.  Discussion: do we need any sort of broader voting body beyond a board.? If so, who could have membership?
4. ### **Project sustainability**:  brainstorming on future funding for the project.
    1.  Offer a hosted version as a subscription, akin to the Home Assistant project model (On github, someone noted that there is already a service doing that, https://trilium.cc/)
    2.  Seek grants or sponsorship from funding organizations
    3.  Accept donations
        1.  Potential platforms suggested: OpenCollective, Github, Patreon
    4.  Issue bounties
        1.  Would need to discuss limitations (e.g.: feature requests that might create other problems for the software, etc)
6. ### **Follow up:**
    1.  Any proposals on having regular, ongoing meetings?

**Comment by root-hal9000** [comment]
I tried to attach a .ics file as a calendar invite, but apparently github does not allow that extension - so attaching here with a fake .png extension in the file :)

![trilium_meeting ics](https://github.com/TriliumNext/Notes/assets/2352828/69332fc4-81be-40ab-9e00-bb10cc879515)

**Comment by CobriMediaJulien** [comment]
@root-hal9000 should we test the infrastructure first on friday?

So everything is going to work?

**Reply by root-hal9000** [reply]
good point - in fact, I think someone already did int he Matrix chat - I will double check and test myself with anyone I find chatting synchronously

**Comment by root-hal9000** [comment]
@TriliumNext/everyone 
Sorry to ping everyone, but just wanted to make sure everyone sees this in case they would like to add anything to the agenda.

**Comment by Alumniminium** [comment]
I'll join.

**Comment by zerebos** [comment]
Were there any notes/meeting minutes taken from the meeting? I was unable to attend as I needed to take my mother to the doctor.

**Reply by meichthys** [reply]
Yes. @root-hal9000 plans to compile the notes and post them soon. I hope all is well with your mother.

**Reply by root-hal9000** [reply]
@rauenzi sorry to hear that ... Yes, I will post them later today

**Reply by lokeshsk** [reply]
I also couldn't join, it would be great if we can have some MoM

**Reply by root-hal9000** [reply]
Sorry for the delays, got busy, then had problems with access on the organization private repo, but it's sorted out. Minutes and invites for future meetings are posted on the internal repository

**Reply by meichthys** [reply]
https://github.com/TriliumNext/.github-private/blob/main/meeting_minutes/2024-07-07-minutes.md

**Comment by TechToch** [comment]
Dear Guys (and ladies),

My I join you as ? I have some experience integrating Trilium on Proxmox/TrueNAS/Docker/ etc.. I would love to try to melt it with Ollama and a RAG system (I just get some learning unit in B3D, deep learning and so on).

See U
XImeng

**Reply by meichthys** [reply]
Welcome!

The next meeting has not yet been scheduled AFAIK.
In the mean time,
Please plug yourself into one of our existing teams (https://github.com/orgs/TriliumNext/teams) and feel free to start contributing.

Also be sure to join our conversations on Matrix: https://matrix.to/#/#triliumnext:matrix.org

**Comment by TechToch** [comment]
PS: I could be the "Note" & "Time" keeper during meetings ;-)
